hack/verify-gofmt.sh

```diff
@@ -38,14 +43,13 @@ find_files() {
         -o -wholename '*/third_party/*' \
         -o -wholename '*/vendor/*' \
         -o -wholename './staging' \
+        -o -wholename '*/bindata.go' \
```

Comment:
We're actually generating `test/e2e/generated/bindata.go`, and I guess it needs gofmt, as in [here](https://github.com/kubernetes/kubernetes/blob/master/hack/generate-bindata.sh#L53)...